src/test/java/com/google/javascript/gents/singleTests/goog_scope.ts

```diff
@@ -1,22 +1,20 @@
 export class Foo {
-  static num: number = 8;
   constructor(public n: number) {}
 
   static foo(): string {
-    return 'this should create a static method on Foo, since it is NOT goog.provided';
+    return 'this is a static method on Foo, since it is NOT goog.provided';
   }
+}
 
-  // TODO: Aggressively export rather than create static methods/fields
-  static qux(): string {
-    return 'this should be directly exported, rather than create a static method, ' +
-        'since it is goog.provided';
-  }
+export function qux(): string {
+  return 'this is directly exported since it is goog.provided';
 }
-const Foo = Foo;
-Foo.Bar = class {
-  static bar(): boolean {
-    return false;
-  }
+
+export const num: number = 8;
```

Comment:
I don't quite understand the behavior here to be honest.  I found the code that does the "aggressive exporting," and disabling it provides a more consistent experience, but breaks a bunch of the existing tests (TODO added).